packages/build/src/fail-on-console-logs.js

```diff
@@ -54,7 +54,8 @@ process.on('exit', code => {
   if (!problems.length) return;
   const log = originalConsole.log;
 
-  log('\n=== ATTENTION - INVALID USAGE OF CONSOLE LOGS DETECTED ===\n');
+  log('\n=== ATTENTION - INVALID USAGE OF CONSOLE LOGS DETECTED ===');
+  log('Learn more at http://ibm.biz/lb-mocha-console-logs\n');
```

Comment:
I think a GitHub permalink be better:

1. Docs are accurate for the specific version of `@loopback/build`

    The docs may change over time to reflect functional changes. This means that the latest version of the docs may not be accurate for the version of `@loopback/build` that is used by the user.

2. The link can be updated when the docs are updated

    This ties in with the first point - when the docs are updated, the link can be updated to the new permalink. This would ensure that the users are looking at the most accurate version of the docs.

Reply:
I prefer a `github` or `loopback.io` link too.

Reply:
+1 to a GitHub link. (Also, the current link adds a single quote at the end for me: https://github.com/strongloop/loopback-next/blob/master/packages/build/README.md#a-note-on-console-logs-printed-by-tests')

Reply:
Thank you for the feedback, I agree with your arguments and will change the link to point to GitHub `master` branch.